lib/Eclair/LLVM/Codegen.hs

```diff
@@ -19,7 +17,7 @@ import Eclair.LLVM.Config
 
 
 llvmSizeOf :: (MonadModuleBuilder m, MonadIO m)
-           => ForeignPtr LibLLVM.Context -> Ptr LibLLVM.TargetData -> Type -> m Word64
```

Comment:
This doesn't show up for me when I do it locally? Why did you add the fully qualified name?

Can we try removing it here (probably by changing the way we import modules in this file)?

Reply:
I think I may have selected the less appropriate option in the IDE when applying the fix. Will remove.